_partners/tim-stallmann.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Tim Stallmann
+slug: tim
```

Comment:
Is slug needed here? I noticed when I changed it locally it didn't have an effect, but I appreciate that it may have some more global purposes that I may be unaware

Reply:
Hah, sorry, merged this without reading your comment first!

Not needed, I'll push another quick commit.